src/engine/sparqlExpressions/AggregateExpression.h

```diff
@@ -61,6 +61,9 @@ class AggregateExpression : public SparqlExpression {
   [[nodiscard]] string getCacheKey(
       const VariableToColumnMap& varColMap) const override;
 
+  // __________________________________________________________________________
+  bool isDistinct() const { return _distinct; }
```

Comment:
While you are at it, you can also add this member function to the `GroupConcat` aggregate, you will use it later anyway.